Copying `atm_beamwidth.txt` to `$DATA` helped, but prepatmiodaobs still bombed.
```
2023-11-15 19:05:43,229 - INFO     - run_bufr2ioda.py: Executing /work2/noaa/da/rtreadon/gdas-validation/global-workflow/sorc/gdas.cd/build/bin/bufr2ioda.x /work2/noaa/da/rtreadon/gdas-validation/comrot/gdas_eval_satwind_JEDI/gdas.20210801/00/obs/atms_2021080100.yaml
PRE-MAIN-INFO BufrParser: Parsing file /work/noaa/rstprod/dump/gdas.20210801/00/atmos/gdas.t00z.atms.tm00.bufr_d
PRE-MAIN-INFO Executing Queries
PRE-MAIN-INFO Building Bufr Data
PRE-MAIN-INFO Exporting Data
 ATMS_Spatial_Average: chechking bt_inout (original) ...
 minval/maxval bt_inout (original) =    133.0100       310.3900
 ATMS_Spatial_Average: minval/maxval scanline =            1        8100
 ATMS_Spatial_Average: chechking bt_inout (remapped) ...
 minval/maxval bt_inout (remapped) =    127.9957      9.9999998E+10
PRE-MAIN-ERROR Exception: Bad parameter: Missing required parameters: sensor. Check your configuration.
terminate called after throwing an instance of 'eckit::BadParameter'
  what():  Bad parameter: Missing required parameters: sensor. Check your configuration.
Traceback (most recent call last):
  File "/work2/noaa/da/rtreadon/gdas-validation/global-workflow/ush/run_bufr2ioda.py", line 94, in <module>
    bufr2ioda(args.current_cycle, args.RUN, args.DMPDIR, args.config_template_dir, args.COM_OBS)
  File "/work2/noaa/da/rtreadon/gdas-validation/global-workflow/ush/python/wxflow/logger.py", line 266, in wrapper
    retval = func(*args, **kwargs)
  File "/work2/noaa/da/rtreadon/gdas-validation/global-workflow/ush/run_bufr2ioda.py", line 79, in bufr2ioda
    cmd()
  File "/work2/noaa/da/rtreadon/gdas-validation/global-workflow/ush/python/wxflow/executable.py", line 230, in __call__
    raise ProcessError(f"Command exited with status {proc.returncode}:", long_msg)
wxflow.executable.ProcessError: Command exited with status -6:
'/work2/noaa/da/rtreadon/gdas-validation/global-workflow/sorc/gdas.cd/build/bin/bufr2ioda.x' '/work2/noaa/da/rtreadon/gdas-validation/comrot/gdas_eval_satwind_JEDI/gdas.20210801/00/obs/atms_2021080100.yaml'

```
Required parameter `sensor` missing.

Who developed the atms converter?   How did they test it?

FYI, we need to increase the wall time for prepatmiodaobs.  10 minutes is no longer sufficient.   Later we can refactor prepatmiodaobs to concurrently run multiple converters via CFP

--- a/parm/ioda/bufr2ioda/atms_beamwidth.txt
+++ b/parm/ioda/bufr2ioda/atms_beamwidth.txt
@@ -1,0 +1,36 @@
+# LOC = data/preproc/atms_beamwidth.dat
+# WMO Satellite ID
+224
+# Version identifier
+1
+# Sampling distance (deg)
+1.11
+# Number of channels to modify
+22 
+# Channel, beam width, new width, cutoff, nxaverage, nyaverage, QC dist
+# Note: to use FFT technique, set new width and cutoff as required and set 
+# nxaverage and nyaverage to 0. To use simple averaging set new width to 0.0
+# and use nxaverage and nyaverage (e.g. 3 3 for 3x3 averaging).
+# QC Dist gives number of points around missing value to flag as missing.
+ 1 5.2 3.33 0.4 0 0 11
+ 2 5.2 3.33 0.4 0 0 11
+ 3 2.2 3.33 0.0 0 0  5
+ 4 2.2 3.33 0.0 0 0  5
+ 5 2.2 3.33 0.0 0 0  5 
+ 6 2.2 3.33 0.0 0 0  5 
+ 7 2.2 3.33 0.0 0 0  5 
+ 8 2.2 3.33 0.0 0 0  5 
+ 9 2.2 3.33 0.0 0 0  5 
+10 2.2 3.33 0.0 0 0  5 
+11 2.2 3.33 0.0 0 0  5 
+12 2.2 3.33 0.0 0 0  5 
+13 2.2 3.33 0.0 0 0  5 
+14 2.2 3.33 0.0 0 0  5 
+15 2.2 3.33 0.0 0 0  5 
+16 2.2 3.33 0.0 0 0  5 
+17 1.1 3.33 0.0 0 0  5
+18 1.1 3.33 0.0 0 0  5
+19 1.1 3.33 0.0 0 0  5
+20 1.1 3.33 0.0 0 0  5
+21 1.1 3.33 0.0 0 0  5
+22 1.1 3.33 0.0 0 0  5

--- a/parm/ioda/bufr2ioda/bufr2ioda_atms.yaml
+++ b/parm/ioda/bufr2ioda/bufr2ioda_atms.yaml
@@ -51,7 +51,6 @@ observations:
           sensorScanAngle:
             sensorScanAngle:
               fieldOfViewNumber: "*/FOVN"
-              heightOfStation: "*/HMSL"
               scanStart: -52.725
               scanStep: 1.110
               sensor: atms
@@ -81,6 +80,7 @@ observations:
               map:
                 _224: npp
                 _225: n20
+#               _226: n21
 
     ioda:
       backend: netcdf

--- a/parm/ioda/bufr2ioda/bufr2ioda_atms.yaml
+++ b/parm/ioda/bufr2ioda/bufr2ioda_atms.yaml
@@ -54,6 +54,7 @@ observations:
               heightOfStation: "*/HMSL"
               scanStart: -52.725
               scanStep: 1.110
+              sensor: atms
 
           sensorChannelNumber:
             query: "*/ATMSCH/CHNM"

--- a/ush/ioda/bufr2ioda/gen_bufr2ioda_yaml.py
+++ b/ush/ioda/bufr2ioda/gen_bufr2ioda_yaml.py
@@ -17,7 +17,7 @@ def gen_bufr_yaml(config, template, output):
     logger.info(f"Using {template} as input")
     bufr_config = parse_j2yaml(template, config)
     # need to do some special manipulation for the splits
-    substitutions = {'splitvar': '{{splits/satId}}'}
+    substitutions = {'splitvar': '{splits/satId}'}
     bufr_config = Template.substitute_structure(bufr_config, TemplateConstants.DOLLAR_PARENTHESES, substitutions.get)
     save_as_yaml(bufr_config, output)
     logger.info(f"Wrote to {output}")

--- a/ush/ioda/bufr2ioda/run_bufr2ioda.py
+++ b/ush/ioda/bufr2ioda/run_bufr2ioda.py
@@ -2,6 +2,7 @@
 import argparse
 import glob
 import os
+import shutil
 from pathlib import Path
 from gen_bufr2ioda_json import gen_bufr_json
 from gen_bufr2ioda_yaml import gen_bufr_yaml
@@ -33,6 +34,10 @@ def bufr2ioda(current_cycle, RUN, DMPDIR, config_template_dir, COM_OBS):
         'PDY': current_cycle.strftime('%Y%m%d'),
         'cyc': current_cycle.strftime('%H'),
     }
+
+    # copy necessary fix files to runtime directory
+    shutil.copy(os.path.join(config_template_dir, "atms_beamwidth.txt"),
+                os.path.join(os.getcwd(), "atms_beamwidth.txt"))
 
     # Specify observation types to be processed by a script
     BUFR_py_files = glob.glob(os.path.join(USH_IODA, 'bufr2ioda_*.py'))

